sdk/trts/trts_veh.cpp

```diff
@@ -251,6 +259,29 @@ static void apply_constant_time_sgxstep_mitigation_and_continue_execution(sgx_ex
         }
     }
 
+    ct_result = ct_decode(&info->cpu_context, &data_address);
+
+    data_tickle_address = stack_tickle_pages & ~0x1;
+    data_tickle_address = cselect64(ct_result, 1, data_address, data_tickle_address);
+    data_tickle_address = cselect64(ct_result, 2, data_address, data_tickle_address);
+    data_tickle_address_is_within_enclave =
+		sgx_is_within_enclave((void*) data_tickle_address, sizeof(uint8_t));
```

Comment:
should be sizeof(uintptr_t)?

Reply:
`sizeof(uint8_t)` is correct. This check is ensuring that the 1-byte read+write in `.ct_check_write` will be safe: https://github.com/intel/linux-sgx/blob/a1eeccba5a72b3b9b342569d2cc469ece106d3e9/sdk/trts/linux/trts_mitigation.S#L325.